consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/MergeProtocolSchedule.java

```diff
@@ -104,11 +103,8 @@ private static ProtocolSpecBuilder applyMergeSpecificModifications(
 
   private static ProtocolSpecBuilder unapplyMergeModificationsFromShanghaiOnwards(
       final ProtocolSpecBuilder specBuilder) {
-    // TODO Withdrawals Get rid of MergeBlockProcessor
     // inherits merge config from MainnetProtocolSpecs.parisDefinition
-    // This would be Function.identify() but MergeBlockProcessor can't be used in
-    // MainnetProtocolSpecs due to circular dependency
-    return specBuilder.blockProcessorBuilder(MergeBlockProcessor::new);
+    return specBuilder;
```

Comment:
Wondering if we should use Function::identity instead now? 
I think it's better not to leave this method around to discourage future changes.

I like that this method has a name though...maybe creating a named FunctionalInterface version of identity() would be a good compromise?

Reply:
I left the function just for the name. A comment would probably just as well though. So will remove it add a comment instead.

Reply:
done